policies/environments/expected.rego

```diff
@@ -45,6 +45,7 @@ expected =
     "oas",
     "oasys",
     "performance-hub",
+    "pra-register",
```

Comment:
Not in alphabetical order like the rest, tiny, but might as well match the rest!

Reply:
resolved with latest PR